src/libraries/Common/src/System/Collections/Generic/LargeArrayBuilder.SizeOpt.cs

```diff
@@ -15,6 +15,11 @@ internal struct LargeArrayBuilder<T>
     {
         private ArrayBuilder<T> _builder; // mutable struct; do not make this readonly
 
+        /// <summary>
+        /// Constructs a new builder.
+        /// </summary>
+        public LargeArrayBuilder() => this = default;
```

Comment:
❔ Considering `_builder = default` is the same behavior, is there any reason why we want this form?

Reply:
I chose it that way as it is _SizeOpt_ and `this = default` has code size _8_ whilst `_builder = default` has code size 13 in IL (according [sharplab](https://sharplab.io/#v2:EYLgtghglgdgNAFxFANgHwAICYCMBYAKAwAYACDHAOgCUBXGBKMAU0oEkHmAnAewAcAytwBuUAMbMAzgG5ChDAGZSkhF1piEpAGI8e+AgG9CpE6T5cowiAmakAgly4QAngCFaqACbdSAfWAeKN5csgSmpMamitq6OAAUAJSkALwAfKQIABZQkimk3gBmELQoCKEAvnJESipqGjE8WIRGYabmlta2Dk5ugcF+AV7coeGRJtE6jYkp6f59Psn5zEUlZYSVBIQA2gKq6ggAMi48tAhxR84nCADSsJ6UAKIAHnwo4lAIcKQCUABetosFFgEgBdeQ1Pb1boudxDLjNdZAA===)) and thought it matters for the _size_.
And (minor) iif in the future any field gets added the build is fine.

On the other hand with
```suggestion
        public LargeArrayBuilder() => _builder = default;
```
(as you suggested) it is more explicit.